esileapclient/tests/unit/common/test_utils.py

```diff
@@ -0,0 +1,78 @@
+import unittest
+from esileapclient.common import utils
+
+
+class TestUtils(unittest.TestCase):
+
+    def setUp(self):
+        self.nodes = [
+            {'properties': {'cpus': '40', 'memory_mb': '131072'}},
+            {'properties': {'cpus': '80', 'memory_mb': '262144'}},
+            {'properties': {'cpus': '20', 'memory_mb': '65536'}},
+        ]
+
+    def test_convert_value(self):
+        self.assertEqual(utils.convert_value('10'), 10)
+        self.assertEqual(utils.convert_value('10.5'), 10.5)
+        self.assertEqual(utils.convert_value('text'), 'text')
+
+    def test_parse_property_filter(self):
+        key, op, value = utils.parse_property_filter('cpus>=40')
+        self.assertEqual(key, 'cpus')
+        self.assertEqual(op, utils.OPS['>='])
+        self.assertEqual(value, 40)
+
+        key, op, value = utils.parse_property_filter('memory_mb<=131072')
+        self.assertEqual(key, 'memory_mb')
+        self.assertEqual(op, utils.OPS['<='])
+        self.assertEqual(value, 131072)
+
+        with self.assertRaises(ValueError):
```

Comment:
Can you use assertRaisesRegex (https://github.com/CCI-MOC/python-esiclient/blob/master/esiclient/tests/unit/v1/test_node_network.py#L460) for these? It normally wouldn't be a huge deal, but the function raises the same error in two different places, so we just want to make sure it's caught by the right one.